specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -3788,6 +3973,156 @@
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BlobInventoryPolicy"
+          },
+          "description": "List of blob inventory policies.",
+          "readOnly": true
+        }
+      },
+      "description": "List of blob inventory policies returned."
+    },
+    "BlobInventoryPolicy": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BlobInventoryPolicyProperties",
+          "x-ms-client-flatten": true,
+          "description": "Returns the storage account blob inventory policy rules."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The storage account blob inventory policy."
+    },
+    "BlobInventoryPolicyProperties": {
+      "properties": {
+        "lastModifiedTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Returns the last modified date and time of the blob inventory policy."
+        },
+        "policy": {
+          "$ref": "#/definitions/BlobInventoryPolicySchema",
+          "description": "The storage account blob inventory policy object. It is composed of policy rules."
+        }
+      },
+      "required": [
+        "policy"
+      ],
+      "description": "The storage account blob inventory policy properties."
+    },
+    "BlobInventoryPolicySchema": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Policy is enabled if set to true."
+        },
+        "destination": {
+          "type": "string",
+          "description": "Container name where blob inventory files are stored. Must be pre-created."
+        },
+        "type": {
+          "type": "string",
+          "description": "The valid value is Inventory",
+          "enum": [
+            "Inventory"
+          ],
+          "x-ms-enum": {
+            "name": "InventoryRuleType",
+            "modelAsString": true
+          }
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BlobInventoryPolicyRule"
+          },
+          "description": "The storage account blob inventory policy rules. The rule is applied when it is enabled."
+        }
+      },
+      "required": [
+        "enabled",
+        "destination",
+        "type",
+        "rules"
+      ],
+      "description": "The storage account blob inventory policy rules."
+    },
+    "BlobInventoryPolicyRule": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Rule is enabled when set to true."
+        },
+        "name": {
```

Comment:
should this always be "default"? Does it means a storage account can only have 1 blob inventory policy?